wolfssl/wolfcrypt/settings.h

```diff
@@ -168,21 +168,7 @@
 
 
 #ifdef COMVERGE
-    #define THREADX
-    #define HAVE_NETX
-    #define WOLFSSL_USER_IO
-    #define NO_WRITEV
-    #define NO_DEV_RANDOM
-    #define NO_FILESYSTEM
-    #define NO_SHA512
-    #define NO_DH
-    /* Allows use of DH with fixed points if uncommented and NO_DH is removed */
-    /* WOLFSSL_DH_CONST */
-    #define NO_DSA
-    #define NO_HC128
-    #define NO_RSA
-    #define NO_SESSION_CACHE
-    #define HAVE_ECC
+    #include <user_settings.h>
```

Comment:
I'd recommend having them define "WOLFSSL_USER_SETTINGS" in their compiler flags. That will include user_settings.h above at ~line 150. Then this entire "COMVERGE" section can be removed.